### DISCUSSION TITLE
How to calculate accuracy of interpretation result from SHAP (like classification problem)

### DISCUSSION THREAD

**Discussion by BinDong314** [shap/shap#2956] [Q&A]
Hi SHAP community,
I had a tabular data with a regression model (i.e., LightGBM) on it.  
Then, we used SHAP to explain the results (e.g., finding negative and positive feature) for new incoming data instances.
Although we can examine the negative and positive feature for a single data instance via some experiments,   these experiments are expensive/time-consuming to run. Lots of involvement of human are also needed to confirm for each instance.  We can not repeat these experiments now. 

However, one user is asking how to get the general accuracy of  the interpretation result from SHAP (like classification problem).
Do anyones can provide some hint or direction we can calculate the general accuracy ?

Thanks in advance,
Bin

**Comment by jainraj** [comment]
My two cents: Do you mean accuracy of SHAP values? If yes, then, with any supervised problem, you would need actual targets. This brings in problem of how do you define these targets (nominal/continuous/ordinal) and how you would measure their departure from calculated SHAP values (i.e., metrics). And because SHAP depends on background dataset too, what is the right dataset to use? and so on.
